google/cloud/aiplatform/training_jobs.py

```diff
@@ -1294,9 +1294,11 @@ def run(
 
         Args:
             model_display_name (str):
```

Comment:
Move this arg docstring down so it's in the same order as the method signature.

Reply:
Will do